packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -363,10 +371,40 @@ class FileCollection extends DocumentCollection {
     if (!href) throw new Error('No related link in server response')
     return this.stackClient.fullpath(href)
   }
-
-  async download(file) {
-    const href = await this.getDownloadLinkById(file._id)
-    forceFileDownload(`${href}?Dl=1`, file.name)
+  /**
+   * Download a file or a specific version of the file
+   *
+   * Passing a filename to forceFileDownload is not enought
+   * for a few browsers since the stack's response header will
+   * not contain that name. Passing the filename to getDownloadLinkBy*
+   * will ask the stack to return this filename in its content-disposition
+   * header response
```

Comment:
It seems this comment is explaning the "how" of this function, I am not sure it belongs to the docstrings as it does not seem to be helpful to the user of this function. I would be in favor of moving the comment above the `if`.

Nit: I thought the start in `getDownloadLinkBy*` was a typo, I think it is clearer if written with `getDownloadLinkBy{Id,Revision}`

Nit: `enought` -> `enought`